perllib/FixMyStreet/Cobrand/CyclingUK.pm

```diff
@@ -236,7 +236,19 @@ sub report_new_munge_before_insert {
     my ($self, $report) = @_;
 
     my $opt_in = $self->{c}->get_param("marketing_opt_in") ? 'yes' : 'no';
-    $report->update_extra_field({ name => 'CyclingUK_marketing_opt_in', value => $opt_in });
+    $report->set_extra_metadata(CyclingUK_marketing_opt_in => $opt_in);
+
+    my @keys = ('CyclingUK_injury_suffered', 'CyclingUK_property_damage', 'CyclingUK_transport_mode', 'CyclingUK_transport_other');
```

Comment:
This leaves `CyclingUK_marketing_opt_in` behind as an empty extra field, guess you need to leave the `update_extra_field` in and add `CyclingUK_marketing_opt_in` to this array?

Reply:
yeah, seems easiest way